server/monitor/src/main/resources/org/apache/accumulo/monitor/templates/tables.ftl

```diff
@@ -87,9 +87,14 @@
           tableList.ajax.reload(null, false ); // user paging is not reset on reload
         }
       </script>
+      <div class="row">
+         <div class="col-xs-12">
+            <h3>Table Overview</h3>
+         </div>
+      </div>
       <div>
-        <table id="tableList" class="table table-bordered table-striped table-condensed">
-          <caption><span class="table-caption">${tablesTitle}</span><br />
```

Comment:
Why were all of the field substitutions (`${}`) replaced with hardcoded values?

Reply:
I put captions on each table and sometimes the 'title' of the page (that was at the top of the page) worked as well for a table caption so I added a different, hardcoded value for the page's overall title. I put the field substitution values back for the pages' titles and just changed the individual tables' captions.